lib/galaxy/tool_util/verify/interactor.py

```diff
@@ -429,7 +429,13 @@ def publish_history(self, history_id: str) -> None:
     def test_data_path(self, tool_id, filename, tool_version=None):
         version_fragment = f"&tool_version={tool_version}" if tool_version else ""
         response = self._get(f"tools/{tool_id}/test_data_path?filename={filename}{version_fragment}", admin=True)
-        result = response.json()
+        try:
+            result = response.json()
+        except Exception as e:
+            raise Exception(
```

Comment:
Can you dump the status code and response.text ? I would prefer this over a guess

Reply:
True, and even our expected error messages are json dumps. Maybe move the response.json to below the statsu code check on the next line ? Then I'm fine with not dumping in the text.